src/averageAges.js

```diff
@@ -14,12 +14,18 @@
  *
  * @return {number}
  */
+
+const getAverageAge = people => people.reduce(
+  (sum, person) => sum + (person.died - person.born), 0
+) / people.length;
```

Comment:
Pay attention to this formatting example 
<img width="619" alt="Screenshot 2023-03-23 at 16 44 11" src="https://user-images.githubusercontent.com/104769168/227239672-b34d623c-4eab-4c4b-aaee-843ebd11b01a.png">